src/posix/platform/radio_spinel.cpp

```diff
@@ -892,6 +892,49 @@ int8_t RadioSpinel::GetRssi(void)
     return rssi;
 }
 
+otError RadioSpinel::GetCoexMetrics(otRadioCoexMetrics *aCoexMetrics)
```

Comment:
use reference instead of pointer, such that we save line 899.

Reply:
Updated